src/locales/cy/fields.yml

```diff
@@ -48,4 +49,7 @@ abandonWarning:
   content: "Ni allwch ddychwelyd i'r sgrin hon os ydych yn chwilio am ffordd arall o brofi eich hunaniaeth."
 
 abandonLink:
-  content: "<a href='/check/abandon' class='govuk-link' id='abandon-link' data-id='abandon'> I do not have my NINO </a>"
+  content: "<a href='/check/abandon' class='govuk-link' id='abandon-link' data-id='abandon'> Nid yw fy rhif Yswiriant Gwladol gennyf </a>"
+
+continueButton:
+  content: "…Parhau"
```

Comment:
Not sure this should have `...` in it?

Reply:
the translations came back and this has now been changed to "Gwirio a pharhau"